src/Tizen.NUI/src/public/BaseComponents/AnimatedVectorImageView.cs

```diff
@@ -0,0 +1,606 @@
+/*
+ * Copyright(c) 2019 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using global::System;
+using global::System.Runtime.InteropServices;
+using System.ComponentModel;
+using tizenlog = Tizen.Log;
+
+namespace Tizen.NUI.BaseComponents
+{
+    /// <summary>
+    /// AnimatedVectorImageView renders an animated vector image
+    /// </summary>
+    // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class AnimatedVectorImageView : ImageView
+    {
+        #region Constructor, Distructor, Dispose
+        /// <summary>
+        /// AnimatedVectorImageView
+        /// </summary>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public AnimatedVectorImageView() : base()
+        {
+            NUILog.Debug($"AnimatedVectorImageView() constructor!");
+            currentStates.url = "";
+            currentStates.frame = -1;
+            currentStates.loopCount = 0;
+            currentStates.loopMode = LoopModes.Forward;
+            //currentStates.stopEndAction = EndActions.Cancel;
+            currentStates.framePlayRangeMin = -1;
+            currentStates.framePlayRangeMax = -1;
+            currentStates.changed = false;
+            currentStates.totalFrame = -1;
+            currentStates.scale = 1.0f;
+        }
+
+        /// <summary>
+        /// AnimatedVectorImageView(float scale)
+        /// </summary>
+        /// <param name="scale"></param>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public AnimatedVectorImageView(float scale) : base()
```

Comment:
I think 'base()' should be changed to 'this()'.

Reply:
yes, you are right. I fixed.